cmd/fs-v1-multipart.go

```
@@ -669,6 +678,20 @@ func (fs fsObjects) CompleteMultipartUpload(bucket string, object string, upload
 		// Allocate staging buffer.
 		var buf = make([]byte, readSizeV1)
 
+		objSize := int64(0)
+		for _, part := range parts {
+			partIdx := fsMeta.ObjectPartIndex(part.PartNumber)
+			if partIdx == -1 {
+				return "", traceError(InvalidPart{})
+			}
+			objSize += fsMeta.Parts[partIdx].Size
+		}
+
+		if objSize > 0 {
+			// Prepare file to avoid disk fragmentation
+			_ = fs.storage.PrepareFile(minioMetaBucket, tempObj, objSize)
+		}
```

Do not ignore the errors.

---

Atleast print it here..

---

> Do not ignore the errors.

yes right, however the xl version of PrepareFile can ignore errors, right?